src/frequenz/sdk/actor/_decorator.py

```diff
@@ -207,10 +207,8 @@ async def _start_actor(self) -> None:
                 except asyncio.CancelledError:
                     logger.debug("Cancelling actor: %s", cls.__name__)
                     raise
-                except Exception as err:  # pylint: disable=broad-except
-                    logger.exception(
-                        "Actor (%s) crashed with error: %s", cls.__name__, err
-                    )
+                except Exception:  # pylint: disable=broad-except
+                    logger.exception("Actor (%s) crashed", cls.__name__)
```

Comment:
it is because pylint prefers this,  and the logging package recommends this approach as the most optimal,  to have the least performance cost especially when log lines get discarded based on how the app is configured,  like debug logs are disabled in production, etc:  https://docs.python.org/3/howto/logging.html#optimization

Reply:
For logging it is recommended to use the built-in (old school) formatting as an *optimization*, as the interpolation will be only done if the message has to be emitted this way. If you use f-strings, interpolation will be done even if you don't end up printing the message. It is also unfortunate that uses old string interpolation with `%s` instead of the new `{}` interpolation, because you have to know both formats. If this optimization is really worth it or not, I don´t know, AFAIK nobody took the time to figure it out.

`pylint` has a rule for this and will complain if you use f-strings.

Reply:
Doing a quick check, it looks like the performance gain is really negligible, so I would be fine with disabling that `pylint` check globally and use f-strings instead.

https://github.com/PyCQA/pylint/issues/2395

Reply:
I created a discussion about it so we avoid polluting this PR, for this PR lets leave the old style until we have a conclusion for the discussion.

* https://github.com/frequenz-floss/frequenz-sdk-python/discussions/206